source/components/molecules/CalendarPicker/CalendarPickerForm.tsx

```diff
@@ -10,7 +10,7 @@ import { colorPalette } from '../../../styles/palette';
 import { PrimaryColor } from '../../../styles/themeHelpers';
 import Button from '../../atoms/Button';
 import Text from '../../atoms/Text';
-import Modal from '../Modal/Modal';
+import { Modal, useModal } from '../Modal';
 
 // Set localized date form.
```

Comment:
väck!

Reply:
Sorry, but could we perhaps focus review comments on code that's actually part of the PR? Otherwise, every  single commit that changes anything in the app will have a whole bunch of style complaints like this, and in view of the quickly approaching deadline, I'm not sure it's a good time investement right now to try and sanitize the entire codebase...

Or perhaps, if you just want to remove superflous comments, then just do it and push it to the branch.